include/deal.II/grid/cell_id.h

```diff
@@ -259,11 +251,15 @@ template <class Archive>
 void
 CellId::serialize(Archive &ar, const unsigned int /*version*/)
 {
+  unsigned int n_child_indices = child_indices.size();
```

Comment:
You don't actually have to serialize this any more since you already serialize the whole `child_indices` array.